src/main/java/frc/robot/RobotContainer.java

```diff
@@ -19,12 +23,17 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
-
+  private final DriveTrain driveTrain = new DriveTrain();
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
+  private final LogitechGamingPad drivePad = new LogitechGamingPad(0);
+  private final JoystickButton buttonA = new JoystickButton(drivePad, 1);
+  private final JoystickButton buttonB = new JoystickButton(drivePad, 2);
```

Comment:
also, since these are only used once, should these be member variables? what if you just instantiated these during `configureButtonBindings`?